api/envoy/api/v2/core/config_source.proto

```diff
@@ -48,6 +49,9 @@ message ApiConfigSource {
   // For REST APIs, the request timeout. If not set, a default value of 1s will be used.
   google.protobuf.Duration request_timeout = 5
       [(validate.rules).duration.gt.seconds = 0, (gogoproto.stdduration) = true];
+
+  // If present, discovery requests made by Envoy will be rate limited.
+  RateLimitSettings rate_limit_settings = 6;
```

Comment:
Shouldn't this be part of `GrpcService` instead, since it doesn't apply to REST (REST already has its own poll period)?

Reply:
As discussed offline, just leaving this as is.